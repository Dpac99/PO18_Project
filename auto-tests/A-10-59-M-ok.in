100010
1
e1.obj
6
6
D1
3
D1
P2
6
D1
3
D1
P2
6
D1
0
0
