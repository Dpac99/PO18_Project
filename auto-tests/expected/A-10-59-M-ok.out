Identificador: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: Ficheiro a abrir: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Nome da disciplina: D1 - P1 (aberto)
D1 - P2 (aberto)
D1 - P4 (aberto)
Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Abrir inquérito: Operação inválida: Problema ao abrir o inquérito do projecto 'P2' da disciplina 'D1'.
Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Nome da disciplina: D1 - P1 (aberto)
D1 - P2 (aberto)
D1 - P4 (aberto)
Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Abrir inquérito: Operação inválida: Problema ao abrir o inquérito do projecto 'P2' da disciplina 'D1'.
Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Nome da disciplina: D1 - P1 (aberto)
D1 - P2 (aberto)
D1 - P4 (aberto)
Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: 